VAMobile/src/components/VABulletList.tsx

```diff
@@ -94,7 +95,14 @@ const VABulletList: FC<VABulletListProps> = ({ listOfText, paragraphSpacing }) =
             <Box mr={20} mt={12}>
               <VAIcon name="Bullet" fill={color || 'bodyText'} height={6} width={6} />
             </Box>
-            <TextView {...textViewProps} accessibilityLabel={a11yLabel || text}>
+            <TextView
+              {...textViewProps}
+              accessibilityLabel={getBulletListTextAccessibilityLabel({
```

Comment:
Instead of generating an a11yLabel by concatenating the various props, would it work to only use the a11yLabel when it's provided, and otherwise not use an a11yLabel at all? If VoiceOver/TalkBack can read the text correctly without an a11yLabel, I think that approach would be simplest. So we'd just have:

```
<TextView {...textViewProps} accessibilityLabel={a11yLabel}>
```

Reply:
Thanks @alexandec in short yes and it was the way I initially approached the change. But changed it after issues with tests. 

It seems that while running snapshot tests the `accessibilityLabel` does not get auto aggregated from the children of the text and causes some tests to fail. so In order for the test to succeed I had 2 options. 

1. Manually generate the label from the various props so that all tests sucesfully assert the values.
2. Modify the Tests that fail to not check using `getByLabelText` as it is no longer fallbacks to `text` 

We can use the simple `<TextView {...textViewProps} accessibilityLabel={a11yLabel}>` if we modify some tests to check for `getByText` instead of `getByLabelText`